.changeset/silver-monkeys-explode.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs-node': patch
+---
+
+Update TechdocsGeneratorExtensionPoint to use GeneratorBase for improved extensibility.
```

Comment:
Can we add an example of this to the docs here: https://github.com/backstage/backstage/edit/master/docs/features/techdocs/extensions.md.